### PR TITLE
Fix for devhub-content deploy

### PR DESCRIPTION
**Pull request by madelinezec**

### CODE REVIEW

File: makefiles/Makefile.devhub-content [hunk 1]
```diff
@@ -62,13 +62,14 @@ next-gen-stage: ## Host online for review
 	@:
 
 next-gen-deploy:	
-	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://docs.mongodb.com --json --all-subdirectories ${ARGS};
-	@echo "Hosted at ${PRODUCTION_URL}/$(filter-out $@,$(MAKECMDGOALS))/index.html";
-	$(MAKE) deploy-search-index
```
Comment (madelinezec) [inline makefiles/Makefile.devhub-content:67]: removed the call to deploy-search-index bc I checked previous version of this makefile and saw the deploy target did not call deploy-search-index.

File: makefiles/Makefile.docs-landing [hunk 2]
```diff
@@ -68,7 +68,7 @@ next-gen-publish: build
 
 next-gen-deploy: build
 	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://docs.mongodb.com --json --all-subdirectories ${ARGS};
-	@echo "Hosted at ${PRODUCTION_URL}/$(filter-out $@,$(MAKECMDGOALS))";
+	@echo "Hosted at ${PRODUCTION_URL}";
```
Comment (madelinezec) [inline makefiles/Makefile.docs-landing:71]: here I just wanted to clean up the echo

File: makefiles/Makefile.devhub-content [hunk 3]
```diff
@@ -62,13 +62,14 @@ next-gen-stage: ## Host online for review
...
+	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://developer.mongodb.com --json --all-subdirectories ${ARGS};
+	@echo "Hosted at ${PRODUCTION_URL}/index.html";
```
Comment (skerschb) [inline makefiles/Makefile.devhub-content:66]: Should deployed url prefix be the  PRODUCTION_URL var?

**Review by skerschb — COMMENTED** [review]
just one nit

File: makefiles/Makefile.devhub-content (hunk 3 above)
Comment (madelinezec) [reply makefiles/Makefile.devhub-content:66]: we can do that - but that is not the case in the other makefiles that I used as a model

Comment (madelinezec) [reply makefiles/Makefile.devhub-content:66]: what would you like to do in this case?

Comment (skerschb) [reply makefiles/Makefile.devhub-content:66]: even docs-landing? This should follow the docs-landing model.

**Pull request merged**